### PR TITLE
CARBONDATA-133 remove show load and delete load command

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
Fixed #https://issues.apache.org/jira/browse/CARBONDATA-133

### CODE REVIEW

File: integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala [hunk 1]
```diff
@@ -1309,7 +1308,7 @@ class CarbonSqlParser()
   }
 
   protected lazy val showLoads: Parser[LogicalPlan] =
-    SHOW ~> (LOADS|SEGMENTS) ~> FOR ~> TABLE ~> (ident <~ ".").? ~ ident ~
+    SHOW ~> (SEGMENTS) ~> FOR ~> TABLE ~> (ident <~ ".").? ~ ident ~
```
Comment (ManoharVanam) [inline integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala]: Change in all test cases

Comment (ManoharVanam) [inline integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala]: ref: https://github.com/HuaweiBigData/carbondata/pull/899/files

**Comment by ravikiran23** [comment]
pls close this PR

**Comment by sujith71955** [comment]
not required since already handled in #899

**Comment by ManoharVanam** [comment]
pls close this PR

**Pull request closed without merging**